packages/device_info/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.4.1
+
+* Support the v2 Android embedder.
```

Comment:
For all places that this term is used, it's the Android "embedding" not the "embedder"